docs/install_guides/installing-openvino-from-archive-macos.md

```diff
@@ -30,60 +30,63 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
 .. tab:: Software Requirements
 
   * `CMake 3.13 or higher <https://cmake.org/download/>`_ (choose "macOS 10.13 or later"). Add `/Applications/CMake.app/Contents/bin` to path (for default install). 
-  * `Python 3.6 - 3.9 <https://www.python.org/downloads/mac-osx/>`_ (choose 3.6 - 3.10). Install and add to path.
-     * Note that OpenVINO is gradually stopping the support for Python 3.6. Python 3.7 - 3.10 are recommended.
+  * `Python 3.6 - 3.10 <https://www.python.org/downloads/mac-osx/>`_ (choose 3.6 - 3.10). Install and add to path.
+     * Note that OpenVINO is gradually phasing out the support for Python 3.6. Python 3.7 - 3.10 are recommended.
   * Apple Xcode Command Line Tools. In the terminal, run `xcode-select --install` from any directory
   * (Optional) Apple Xcode IDE (not required for OpenVINO™, but useful for development)
 
+.. note::
+   The package may also be installed on other versions of macOS, but only the specific versions listed here are fully validated.
 @endsphinxdirective
 
 ## Installing OpenVINO Runtime
 
 ### <a name="install-core"></a>Step 1: Install OpenVINO Core Components
 
-1. Select and download the OpenVINO™ archive files from [Intel® Distribution of OpenVINO™ toolkit for macOS download page](https://software.intel.com/en-us/openvino-toolkit/choose-download/free-download-macos). There are typically two files for you to download: 
+1. Open a command prompt terminal window. 
+2. Create the `/opt/intel` folder for OpenVINO and switch to it by using the following commands. If the folder already exists, skip the `mkdir` command.
    ```sh
-   m_openvino_toolkit_<operating system>_<release version>_<package ID>_x86_64.tgz
-   m_openvino_toolkit_<operating system>_<release version>_<package ID>_x86_64.tgz.sha256
-   ``` 
-   where the `.sha256` file is used to verify the success of the download process.
-
-2. Locate the downloaded files in your system. This document assumes the files are in your `Downloads` directory. 
+   sudo mkdir /opt/intel
+   cd /opt/intel
+   ```
+   > **NOTE**: The `/opt/intel` path is the recommended folder path for installing OpenVINO. You may use a different path if desired.
 
-3. Open a command prompt terminal window, and verify the checksum of the `sha256` file by using the following command:
+3. Download the OpenVINO Runtime archive file for macOS from the [OpenVINO releases page](https://github.com/openvinotoolkit/openvino/releases/tag/2022.2.0), extract it, and rename the extracted folder to `openvino_2022.2.0.7713` by using the following commands:
    ```sh
-   shasum -c -a 256 <archive name>.tgz.sha256
+   sudo curl -L https://github.com/openvinotoolkit/openvino/releases/download/2022.2.0/m_openvino_toolkit_osx_2022.2.0.7713.af16ea1d79a_x86_64.tgz --output openvino_2022.2.0.7713.tgz
+   sudo tar -xf openvino_2022.2.0.7713.tgz
+   sudo mv m_openvino_toolkit_osx_2022.2.0.7713.af16ea1d79a_x86_64 openvino_2022.2.0.7713
```

Comment:
can we use first two commands w/o sudo and only move from ~/Downloads to /opt with sudo?